crypto/fipsmodule/bcm_o/AL2_x86_64/gcc-7/shared/README.md

```diff
@@ -0,0 +1,118 @@
+### Generate bcm.o commands
```

Comment:
Now that we removed bcm.o and the hash from this folder what do you think about moving this up to the fipsmodule folder and name it something like REUSE_BCM.md or VENDOR_AFFIRMATION.md?

Reply:
Will change.

Reply:
Added `crypto/fipsmodule/VENDOR_AFFIRM.md`